### ISSUE TITLE
Small issue: always loads last used checkpoint

### ISSUE THREAD

**Issue by Taikakim** [TheLastBen/fast-stable-diffusion#1332]
Despite defining otherwise in the Path_to_MODEL, the notebook always loads the last used model when launching, I have to select the right model again in the GUI. This has been going on for some time now, version is from yesterday or so.

**Comment by TheLastBen** [comment]
you're probably using a path to a folder instead of the full path to the ckpt

**Comment by Taikakim** [comment]
I'm taking the "copy path" from the file browser, and pasting that to the field, isn't this the format it's expecting?

**Comment by TheLastBen** [comment]
does the path contain the ckpt file ?

**Comment by Taikakim** [comment]
Yes. When I run the notebook for the absolute first time, it picks the ckpt correctly. This is a bit puzzling. Since I "copy path" directly from the notebook file browser, all character cases etc should be correct.

**Comment by TheLastBen** [comment]
if you don't specify which model to load, it will load the first model alphabetically in the models folder.
